packages/experiments/src/components/Pagination/PageNumber.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { IPageNumberProps } from './PageNumber.types';
+
+export class PageNumber extends React.Component<IPageNumberProps, {}> {
+  constructor(props: IPageNumberProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const { className, ariaLabel, page, selected } = this.props;
+
+    return (
+      <button
+        key={page}
+        className={className}
```

Comment:
Thinking that you can use `getNativeProps` here to wrap a lot of existing button props.

Reply:
I tried it and look like the only property I was able to get rid of is the className. I am not very familiar with getNavtiveProps, please let me know if I am not using it in the correct way 

---
In reply to: [233569584](https://github.com/OfficeDev/office-ui-fabric-react/pull/7100#discussion_r233569584) [](ancestors = 233569584)